vc/vc.go

```diff
@@ -32,6 +32,15 @@ type DataModel[T CredentialSubject] struct {
 	IssuanceDate      string             `json:"issuanceDate"`               // https://www.w3.org/TR/vc-data-model/#issuance-date
 	ExpirationDate    string             `json:"expirationDate,omitempty"`   // https://www.w3.org/TR/vc-data-model/#expiration
 	CredentialSchema  []CredentialSchema `json:"credentialSchema,omitempty"` // https://www.w3.org/TR/vc-data-model-2.0/#data-schemas
+	Evidence          []Evidence         `json:"evidence,omitempty"`         // https://www.w3.org/TR/vc-data-model/#evidence
```

Comment:
```suggestion
	Evidence          []any         `json:"evidence,omitempty"`         // https://www.w3.org/TR/vc-data-model/#evidence
```

Reply:
hmm, seemed from the issue https://github.com/TBD54566975/web5-go/issues/79 that we do want to make Type required, which means we can't have `any` here, right?

Reply:
this is true - when we have concrete types of evidence. until then we would be making the evidence property useless since we would not be able to add properties to an evidence property besides `id` or `type`